submissions/krylenger/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,1342 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body class="body">
+    <main class="container-half-screen container-half-screen--film-info">
+      <header class="film-info__header">
+        <h1 class="film-info__title">eX machina</h1>
+        <h2 class="film-info__subtitle">Drama, Sci-Fi, Thriller</h2>
+        <h2 class="film-info__rate">9.4</h2>
+      </header>
+      <section class="film-info__main">
+        <h6 class="film-info__description">
+          A young programmer is selected to participate in a ground-breaking experiment in synthetic intelligence by
+          evaluating the human qualities of a highly advanced humanoid A.I.
+        </h6>
+      </section>
+    </main>
+    <section class="container-half-screen container-half-screen--grey">
+      <form class="container-half-screen--choose-section" action="#">
+        <header class="choose-section__header">
+          <button
+            class="choose-section__header-button choose-section__header-button--back"
+            aria-label="Go Back"
+          ></button>
+          <h2 class="choose-section__title">Step 3/3</h2>
+          <button
+            class="choose-section__header-button choose-section__header-button--filters"
+            aria-label="Settings"
+          ></button>
+        </header>
+        <section class="choose-section__main">
+          <h5 class="choose-section__main-title">
+            <em class="text-italic"
+              >Please choose <span class="text-bold">date</span>, <b class="text-bold">time</b> and
+              <b class="text-bold">seat</b>.</em
+            >
+          </h5>
+          <div class="choose-section__time-container">
+            <label class="choose-section__time">
+              <input class="choose-section__input visually-hidden" type="radio" value='A' name="date" id="dateMon" />
+              <span class="choose-section__span--time">
+                <time datetime="2021-02-01" class="choose-section__time--bold">Mon</time>
+                <time datetime="2021-02-01" class="choose-section__time-number">1</time>
```

Comment:
Better use one `time` tag and two `span` inside this tag.
Also, don't forget about class naming (line 48).

Reply:
✅ Done